matrix_sdk/src/room/common.rs

```diff
@@ -146,6 +147,23 @@ impl Common {
         self.client.send(request, None).await
     }
 
+    /// Sends a request to `/_matrix/client/r0/rooms/{roomId}/event/{eventId}`
+    /// and returns a `get_room_event::Response` that contains a event
+    /// (`AnyRoomEvent`).
+    pub async fn event(
+        &self,
+        request: impl Into<get_room_event::Request<'_>>,
+    ) -> Result<RoomEvent> {
+        let request = request.into();
+        let event = self.client.send(request, None).await?.event.deserialize()?;
```

Comment:
I wonder if we should return the event as is if we fail to deserialize here, users could try to deserialize into something else and possibly retry decryption if we do so.

Probably not that important though.